examples/diagnostics_and_criticism/sampler-stats.ipynb

```diff
@@ -14,29 +14,40 @@
   },
```

Comment:
<p>Sorry, it should be <code>list(trace.sample_stats.data_vars)</code> we only want to see the names here</p><p> </p><p>EDIT: actually scratch that, we should use that below with the multiple samples, but show the html repr of <code>trace.sample_stats</code>  here so we can then comment on both variable names _and attributes!_ There is nothing on attributes because there did not use to be here. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc-examples/pull/95/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
<p>so for the first sampling we go for, <code>trace.sample_stats</code>, followed by markdown description?</p>